Indicators/WilliamsPercentR.cs

```diff
@@ -0,0 +1,94 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    /// Williams %R, or just %R, is the current closing price in relation to the high and low of
+    /// the past N days (for a given N). The value of this indicator fluctuats between -100 and 0. 
+    /// The symbol is said to be oversold when the oscillator is below -80%,
+    /// and overbought when the oscillator is above -20%. 
+    /// </summary>
+    public class WilliamsPercentR : TradeBarIndicator
+    {
+        /// <summary>
+        /// Gets the Max indicator
+        /// </summary>
+        public Maximum Max { get; private set; }
+
+        /// <summary>
+        /// Gets the Min indicator
+        /// </summary>
+        public Minimum Min { get; private set; }
```

Comment:
Specifically the property name, but we can also update the comment as well, thanks!